syntax = "proto3";

service ClientCallService{
rpc sendPartitionFile(File) returns(Request) {}

}

message File {
    repeated bytes content = 1;
}

message Request {
    string requestFile = 2;
}